File: okj/szoftverfejleszto/vizibicikli/java/Kolcsonzes.java
import java.time.*;

public class Kolcsonzes {

    public final String nev;
    public final String jarmuAzonosito;
    public final LocalTime elvitelIdopont;
    public final LocalTime visszahozatalIdopont;

    public Kolcsonzes(String line) {
        var split = line.split(";");

        nev = split[0];
        jarmuAzonosito = split[1];
        elvitelIdopont = LocalTime.of(Integer.parseInt(split[2]), Integer.parseInt(split[3]));
        visszahozatalIdopont = LocalTime.of(Integer.parseInt(split[4]), Integer.parseInt(split[5]));
    }
}

File: okj/szoftverfejleszto/vizibicikli/c#/Kolcsonzes.cs
using System;

public class Kolcsonzes {

    public readonly string nev;
    public readonly string jarmuAzonosito;
    public readonly TimeSpan elvitelIdopont;
    public readonly TimeSpan visszahozatalIdopont;

    public Kolcsonzes(string line) {
        var split = line.Split(';');

        nev = split[0];
        jarmuAzonosito = split[1];
        elvitelIdopont = new TimeSpan(int.Parse(split[2]), int.Parse(split[3]), 0);
        visszahozatalIdopont = new TimeSpan(int.Parse(split[4]), int.Parse(split[5]), 0);
    }
}